pom.xml

```diff
@@ -203,6 +204,56 @@
 									</excludes>
 								</filter>
 							</filters>
+							<relocations>
```

Comment:
You look sharded everything. Then if we add a new library, you want to shaded as well right? 
Then it might be better to add the direction to the developer's guide.